### DISCUSSION TITLE
Can we use external editor for masking?

### DISCUSSION THREAD

**Discussion by Renaldas111** [hlky/stable-diffusion-webui#1021] [General]
Masking in webui is quite inconvenient, as the brush is very small and there is no possibility to do precise masking. Can we create a mask in photoshop-like external editor and give it to the neural network to make an inpainting? Even if it is working with command line, no problem.
Thanks for help!

**Comment by codedealer** [comment]
Masking in alpha channel is supported since #921 but it is on `dev` branch currently. Either wait till it's merged to `master` at some point in the future or update to `dev` with the note that it's unstable and can have bugs.
